config/openmp.m4

```diff
@@ -96,7 +96,7 @@ AC_DEFUN([AX_HAVE_SYNC_FETCH_AND_ADD],
 		[ have_sync_fetch_and_add="yes" ]
 	)
 
-	if test "${have_sync_fetch_and_add}" = "yes" ; then
+	if test "${have_sync_fetch_and_add}" = "yes" -a "${IS_GR740_MACHINE}" = "no" ; then
```

Comment:
Doesn't checking for __sync_fetch_and_add work on GR740?

Reply:
I believe that the __sync_fetch_and_add was defined in the Gr740 toolchain but is was empty (stub) or not working properly.